### PR TITLE
Implement JWT token authorization

### PR DESCRIPTION
**Pull request by dmytrostriletskyi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,9 +2,57 @@
...
+    "token": "eyJ0e....eyJ1c2VyX2....sOx4S9zpC..."
+}
+```
+
+* `POST | /authentication/token/verification/` - check if `JWT token` token is valid.
+
+```bash
+$ curl -v -X POST -H "Content-Type: application/json" -d '{"token":"eyJ0e....eyJ1c2VyX2....sOx4S9zpC..."}' \
+      http://localhost:8000/authentication/token/verification/ | python -m json.tool
+
```
Comment (anastasiia-bilova) [inline README.md]: Remove the blank line.

File: directory/urls.py [hunk 2]
```diff
@@ -1,21 +1,25 @@
...
+    obtain_jwt_token,
+    refresh_jwt_token,
+    verify_jwt_token,
+)
+
+authentication_endpoints = [
+    path('token/obtaining/', obtain_jwt_token),
+    path('token/refreshing/', refresh_jwt_token),
+    path('token/verification/', verify_jwt_token),
+
```
Comment (anastasiia-bilova) [inline directory/urls.py]: Remove the blank line.

Comment (dmytrostriletskyi) [reply directory/urls.py]: @anastasiia-bilova, done.

File: README.md (hunk 1 above)
Comment (dmytrostriletskyi) [reply README.md]: @anastasiia-bilova, done.

**Pull request merged**